src/config.ts

```diff
@@ -101,16 +102,16 @@ const CIE_METADATA_URL = getRequiredENVVar("CIE_METADATA_URL");
 export const STARTUP_IDPS_METADATA:
   | Record<string, string>
   | undefined = fromNullable(process.env.STARTUP_IDPS_METADATA)
-  .map(_ =>
-    parseJSON(_, toError)
-      .chain<Record<string, string> | undefined>(_1 =>
-        STRINGS_RECORD.decode(_1).mapLeft(
-          err => new Error(errorsToReadableMessages(err).join(" / "))
+    .map(_ =>
+      parseJSON(_, toError)
+        .chain<Record<string, string> | undefined>(_1 =>
+          STRINGS_RECORD.decode(_1).mapLeft(
+            err => new Error(errorsToReadableMessages(err).join(" / "))
+          )
         )
-      )
-      .getOrElse(undefined)
-  )
-  .getOrElse(undefined);
+        .getOrElse(undefined)
+    )
```

Comment:
it looks like you have an odd configuration of prettier, can you revert these changes?